src/main/java/seedu/address/logic/commands/EditPropertyCommand.java

```diff
@@ -44,17 +44,17 @@
     public static final String COMMAND_WORD = "edit property";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits a property in the app. \n"
-            + "Parameters: INDEX "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_TYPE + "TYPE "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + PREFIX_POSTAL + "POSTAL "
-            + PREFIX_DEADLINE + "DEADLINE "
-            + PREFIX_REMARK + "REMARK "
-            + PREFIX_CLIENT_NAME + "CLIENT_NAME "
-            + PREFIX_CLIENT_CONTACT + "CLIENT_CONTACT "
-            + PREFIX_CLIENT_EMAIL + "CLIENT_EMAIL "
-            + PREFIX_CLIENT_ASKING_PRICE + "CLIENT_ASKING_PRICE \n"
+            + "Parameters: INDEX ["
+            + PREFIX_NAME + "NAME] ["
+            + PREFIX_TYPE + "TYPE] ["
+            + PREFIX_ADDRESS + "ADDRESS] ["
+            + PREFIX_POSTAL + "POSTAL] ["
+            + PREFIX_DEADLINE + "DEADLINE] ["
+            + PREFIX_REMARK + "REMARK] ["
+            + PREFIX_CLIENT_NAME + "CLIENT_NAME] ["
+            + PREFIX_CLIENT_CONTACT + "CLIENT_CONTACT] ["
+            + PREFIX_CLIENT_EMAIL + "CLIENT_EMAIL] ["
+            + PREFIX_CLIENT_ASKING_PRICE + "CLIENT_ASKING_PRICE]\n"
```

Comment:
Just a suggestion: I will suggest to format the square brackets like this so that it is much neater 

![image](https://user-images.githubusercontent.com/65358857/113694837-d2f60780-9702-11eb-9b2c-c08c439e5be9.png)

Reply:
oops i was being too lazy, changed it!